.github/workflows/rollback-console.yml

```diff
@@ -0,0 +1,84 @@
+name: 'Rollback console'
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version that should be set on rollback'
+        required: true
+        type: string
+
+jobs:
+  rollback:
+    runs-on: ubuntu-22.04
+    timeout-minutes: 25
+    steps:
+      - name: Set up QEMU
```

Comment:
Do we need Quemu for that? We do not build multi-platform here. If we have multiplatform docker image We should retag all of them. Docker does not support multiplatform itself does it? I think it is required to use dockerx for multiplatform. But for docker we also do not need quemu